app/controllers/events/view/edit/other-details.js

```diff
@@ -0,0 +1,31 @@
+import classic from 'ember-classic-decorator';
+import { action } from '@ember/object';
+import Controller from '@ember/controller';
+import EventWizardMixin from 'open-event-frontend/mixins/event-wizard';
+
+@classic
+export default class OtherDetailsController extends Controller.extend(EventWizardMixin) {
+  @action
+  save() {
+    this.saveEventDataAndRedirectTo(
+      'events.view.index',
+      ['tickets', 'socialLinks', 'copyright', 'tax', 'stripeAuthorization']
+    );
+  }
+
+  @action
+  async move(direction, data) {
```

Comment:
Codacy found an issue: ['data' is defined but never used.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6089057)